bindings/csharp/Swig/cntk_cs.i

```diff
@@ -1274,6 +1274,55 @@
         return;
     }
 
+	public void CopyVariableValueTo<T>( Variable outputVariable, 
```

Comment:
Could you please change the method name to GetSparseData<T>(....), to align with those for dense and onehot. The CopyVariableValueTo for desne and onehot was deprecated now.